subscribe.js

```diff
@@ -0,0 +1,40 @@
+const silo = require('./combineNodes.js').silo;
+
+const subscribe = (component, name) => {
```

Comment:
The subscribe function should be a method on silo, as to not require the importing of the function, and the creation of multiple Silos.